icicle/include/icicle/program/symbol.h

```diff
@@ -10,7 +10,8 @@ namespace icicle {
    *
    */
   enum ProgramOpcode {
-    OP_ADD = 0,
+    OP_COPY = 0,
```

Comment:
suggestion: COPY -> MOVE
I think this is the name usually used but not really important

Reply:
it does not delete the origin so... copy